src/logic/operate.js

```diff
@@ -0,0 +1,16 @@
+import Big from 'big.js';
+
+const operate = (numberOne, numberTwo, operation) => {
+  const num1 = new Big(numberOne);
+
```

Comment:
Kindly wrap the second number with `Big` class as you did with the num1. So for example; add `const num2 = new Big(numberTwo)` as raised in the previous review